server/tests/api_v4/authenticated/test_block_create.py

```diff
@@ -27,6 +31,28 @@ async def test_authenticated_block_create_ok(coolorg: CoolorgRpcClients, backend
     assert dump == expected_dump
 
 
+async def test_authenticated_block_create_realm_not_found(
+    coolorg: CoolorgRpcClients, backend: Backend
+) -> None:
+    block_id = BlockID.new()
+    block = b"<block content>"
+
+    # Try to create block with an invalid realm ID
+    rep = await coolorg.alice.block_create(block_id, VlobID.new(), block)
+    assert rep == authenticated_cmds.v4.block_create.RepRealmNotFound()
+
+    # Block should not exist
+    content = await backend.block.read_as_user(
+        organization_id=coolorg.organization_id,
+        author=coolorg.alice.device_id.user_id,
+        block_id=block_id,
+    )
+    assert content != BlockReadBadOutcome.BLOCK_NOT_FOUND.value
```

Comment:
This seems strange to me: `read_as_user` is supposed to return a `BlockReadBadOutcome` enum, and we expect it to be abound the block not being found...

```suggestion
    assert content = BlockReadBadOutcome.BLOCK_NOT_FOUND
```